overlord/devicestate/devicestate_test.go

```diff
@@ -3252,10 +3252,8 @@ func (s *deviceMgrSuite) TestUpdateGadgetOnClassicErrorsOut(c *C) {
 
 	s.state.Unlock()
 
-	for i := 0; i < 6; i++ {
-		s.se.Ensure()
-		s.se.Wait()
-	}
+	// we could use:
+	s.o.Settle(0.5 * time.Second)
```

Comment:
vet complains:
```
overlord/devicestate/devicestate_test.go:3256:17: constant 0.5 truncated to integer
```

Reply:
FWTW the timeout here is really relevant if settle is not converging (e.g. we have a real bug in code or test), there is no real benefit with fractions of seconds here, it's fine to have large timeout; it might actually be better to have a few seconds here (we seem to be using 5*Second in many places) to avoid failures on very slow boards.